### DISCUSSION TITLE
LegacyDefaultComboCounter shows the wrong combo value

### DISCUSSION THREAD

**Discussion by nuewink** [ppy/osu#29888] [Q&A] [answered]
Hi. I have a "max combo" object in my skin, and it often shows a different value to the combo object. I don't know which one shows the wrong number, but it was really irritating to see this happen.
I noticed that if I put an "argon" or "defaultcombocounter" instead of the "legacy combo counter", they will show the same value as the max combo, see the video below. Is this a bug? I don't mind using the lazer combo counter, in fact I will use it now, but I decided to post about it anyway

https://github.com/user-attachments/assets/5079134c-ae48-4de4-bddf-f9acd07203d5

**Comment by nuewink** [comment]
Or maybe all of the Lazer combo counters are wrong, and it's the legacy one that shows the correct value. I have no idea

**Comment by bdach** [comment, marked as answer]
The legacy combo counter is bugged. The fix is already slated for next release whenever it happens.
